### PR TITLE
NOTIF-145 Fix lib folder

### PR DESCRIPTION
**Pull request by theute**

### CODE REVIEW

**Review by gwenneg — CHANGES_REQUESTED** [review]
Wait, I think we need more than that.

**Comment by spinder** [comment]
lgtm

File: src/main/docker/Dockerfile-build.jvm [hunk 1]
```diff
@@ -37,7 +37,7 @@ RUN microdnf install openssl curl ca-certificates ${JAVA_PACKAGE} \
 # Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 
-COPY --from=build /home/jboss/target/lib/* /deployments/lib/
+COPY --from=build /home/jboss/target/quarkus-app/lib/* /deployments/lib/
 COPY --from=build /home/jboss/target/quarkus-app/quarkus-run.jar /deployments/app.jar
```
Comment (gwenneg) [inline src/main/docker/Dockerfile-build.jvm:41]:
```suggestion
COPY --from=build /home/jboss/target/quarkus-app/* /deployments/
```

Comment (gwenneg) [reply src/main/docker/Dockerfile-build.jvm:41]: From the Quarkus migration guilde:

> When deploying your application, make sure to deploy the entire quarkus-app directory.

**Review by gwenneg — APPROVED** [review]
LGTM

**Comment by theute** [comment]
Wait that won't work neither :)

**Comment by gwenneg** [comment]
`run-java.sh` doesn't seem to be expecting a particular JAR file name.

**Pull request merged**